pkg/controller/installconfig/installconfig_controller.go

```diff
@@ -194,14 +206,30 @@ func (r *ReconcileInstallConfig) ensureCheSubscription(logger logr.Logger, ns st
 	return subCreated, err
 }
 
-func (r *ReconcileInstallConfig) StatusUpdate(logger logr.Logger, installConfig *v1alpha1.InstallConfig, statusUpdater func(installConfig *v1alpha1.InstallConfig, message string) error, msg string) (reconcile.Result, error) {
+func (r *ReconcileInstallConfig) ensureTektonSubscription(logger logr.Logger, ns string, installConfig *v1alpha1.InstallConfig) (bool, error) {
+	tektonSub := tekton.NewSubscription(ns)
+	subCreated := false
```

Comment:
we can remove this variable .. and return true/false appropriately so it will become explicit .. whether t/f is returned.

Reply:
I would keep this the same way to have DRY